bikeshed/config/printjson.py

```diff
@@ -79,4 +79,4 @@ def printjsonprimitive(x):
         return "null"
     if isElement(x):
         return repr(x) + ":" + outerHTML(x)
-    raise Exception(x)
+    raise ValueError(f"Could not print value: {x}")
```

Comment:
Not sure if it's better to have a broad Exception here

Reply:
Nah, seems fine, I was just lazy about summoning the correc type.